.vscode/launch.json

```diff
@@ -78,5 +78,25 @@
         "TS_NODE_TRANSPILE_ONLY": "true",
       },
     },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Only Migrate",
+      "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
+      "runtimeArgs": ["--nolazy", "-r", "ts-node/register"],
+      "args": [
+        "--no-timeout",
+        "${workspaceFolder}/packages/migrate/test/migrate.test.ts"
+      ],
+      "sourceMaps": true,
+      "cwd": "${workspaceFolder}/packages/migrate",
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen",
+      "protocol": "inspector",
```

Comment:
I think this property is deprecated?

Reply:
Listed here: https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_javascript-debug-terminal
But listed with attributes for 'attach'. So removing it.